lib/active_model_serializers/key_transform.rb

```diff
@@ -66,5 +66,26 @@ def underscore(value)
     def unaltered(value)
       value
     end
+
+    # Transforms string to selected case
+    # Accepts string in any case: 'camel', 'undescore', 'dashed'.
+    #
+    # @example:
+    #   transform('SomeClass', :underscore) => 'some_class'
+    #   transform('some_class', :snake) => 'someClass'
+    #   etc...
+    def transform(string, type)
+      string = string.underscore
+      case type.to_sym
+      when :dashed
+        string.dasherize
+      when :camel
```

Comment:
I may have brought this up with the original key_transform PR, but `camelize` does `PascalCase` where as `camelize(:lower)` is `camelCase`